mu4/notation/internal/notationconfiguration.cpp

```diff
@@ -105,6 +105,21 @@ Channel<QColor> NotationConfiguration::backgroundColorChanged() const
     return m_backgroundColorChanged;
 }
 
+QColor NotationConfiguration::pageColor() const
+{
+    return QColor("#ffffff");
```

Comment:
I don't remember if whether these properties are depending on some "style" or "preferences" values. Have you ckecked it already?

Reply:
At this moment they don't depend on style or preferences